Contest_API.md

```diff
@@ -331,8 +331,6 @@ access to.
 | [admin_submit](#modifying-submissions)    | Submit as an admin                           |
 | [admin_clar](#modifying-clarifications)   | Submit clarifications as an admin            |
 
-TODO - add capabilities related to team view, awards, and freeze time.
```

Comment:
Why remove this TODO? We have not actually done it yet.

Reply:
FYI, this is a literal cherry-pick of https://github.com/icpc/ccs-specs/pull/130, should it also not have been removed there?